.github/workflows/be_review_prs.yml

```diff
@@ -104,15 +103,10 @@ jobs:
             echo "approval_status=confirmed" >> $GITHUB_OUTPUT
           fi
 
-      - name: Comment PR - Approval Required
-        if: failure() && steps.verify_approval.outputs.approval_status == 'required' && steps.find-required-comment.outputs.comment-id == ''
-        uses: peter-evans/create-or-update-comment@v4
-        with:
-          issue-number: ${{ github.event.pull_request.number }}
-          body: "This PR requires approval from backend-review-group. Please add backend-review-group as reviewers (if they're not already listed)."
-
       - name: Comment PR - Approval Confirmed
-        if: success() && steps.verify_approval.outputs.approval_status == 'confirmed'
+        if: |
+          success() && steps.verify_approval.outputs.approval_status == 'confirmed' &&
+          steps.find_backend_approval_comment.outputs.comment-id != ''
```

Comment:
Make sure the Comment PR - Approval Confirmed comment does not exist before adding this again